pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -26,16 +26,18 @@ Please add breaking changes here when merged to the `develop` branch.
   - Rename `setMeshTriangleWithEdges` to `setMeshTriangle` and `setMeshQuadWithEdges` to `setMeshQuad`. The edge-based implementation was removed.
   - Use the new bulk functions to reduce sanitization overhead: `setMeshEdges`, `setMeshTriangles`, `setMeshQuads`, `setMeshTetrahedra`
 - Remove `mapWriteDataFrom()` and `mapReadDataTo()`.
-- Remove `initializeData()` and initialize the data after defining the mesh and before calling `initialize()` if `requiresInitialData()` is `true`.
+- Remove `initializeData()`. The functions `initializeData()` and `ìnitialize()`have been grouped in the new function `initialize()`. Before calling `ìnitialize()` you have to initialize the mesh and the data ( if `requiresInitialData()` is `true`).
```

Comment:
Minor language tweaking (more precise term, comma to split time from the rest of the sentence, and a missing space in Markdown):

```suggestion
- Remove `initializeData()`. The functions `initializeData()` and `ìnitialize()` have been merged into the new function `initialize()`. Before calling `ìnitialize()`, you have to initialize the mesh and the data ( if `requiresInitialData()` is `true`).
```